app/src/main/java/com/android/streetworkapp/model/image/ImageModels.kt

```diff
@@ -0,0 +1,52 @@
+package com.android.streetworkapp.model.image
+
+import android.net.Uri
+import com.google.firebase.Timestamp
+
+/**
+ * A data class that represents a collection of images related to a park.
+ *
+ * @property id The id of the collection.
+ * @property images A list of ParkImage
+ */
+data class ParkImageCollection(
+    val id: String,
+    val images: List<ParkImageDatabase>
+) // note: if images field were to be changed, make sure to match the field name change in
+// ImageRepositoryFirestore::uploadImage
+
+/**
+ * A data class that represents an image related to a park in the database, uploaded by a user, and
+ * associated with a rating.
+ *
+ * @property imageB64 The image in Base64 string format.
+ * @property userId The id of the user who uploaded the image.
+ * @property rating A pair of integers representing the rating, where the first element represents
+ *   all the positive ratings and the second the negative ones.
```

Comment:
Question: I am not sure I understand why the property rating in contains "the first element represents
 all the positive ratings and the second the negative ones", could you explain this choice?

Reply:
basically it's Pair(#the number of ppl who liked this picture, #the number of ppl who disliked this picture)

Reply:
I now see better were it is going, thank you!